models.py

```diff
@@ -338,3 +339,61 @@ def get_peaks_for_display():
 
     def __repr__(self):
         return 'Variable:' + self.id
+
+
+class RwrAccountType(Enum):
+    INVASION = 'INVASION'
+    PACIFIC = 'PACIFIC'
+
+
+class RwrAccount(db.Model):
+    __tablename__ = 'rwr_accounts'
+    __table_args__ = (db.Index('type_username_idx', 'type', 'username'), )
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+
+    type = db.Column(db.Enum(RwrAccountType), nullable=False)
+    username = db.Column(db.String(16), nullable=False)
+    created_at = db.Column(ArrowType, default=arrow.utcnow().floor('minute'), nullable=False)
+    updated_at = db.Column(ArrowType, default=arrow.utcnow().floor('minute'), nullable=False)
+
+    @memoized_property
+    def stats(self):
+        return RwrAccountStat.query.filter(RwrAccountStat.rwr_account_id == self.id).all()
+
```

Comment:
Add a memoized method to know if this RWR account has stats or not (COUNT SQL query)